e2e/process-services-cloud/people-group-cloud-component.e2e.ts

```diff
@@ -0,0 +1,141 @@
+import { browser } from 'protractor';
+import { Identity } from '../actions/APS-cloud/identity';
+import { GroupIdentity } from '../actions/APS-cloud/groupIdentity';
+import CONSTANTS = require('../util/constants');
+
+describe('People Groups CLoud Component', () => {
+
+    describe('People Groups CLoud Component', () => {
+        const settingsPage = new SettingsPage();
+        const loginSSOPage = new LoginSSOPage();
+        const navigationBarPage = new NavigationBarPage();
+        const peopleGroupCloudComponentPage = new PeopleGroupCloudComponentPage();
+        const peopleCloudComponent = new PeopleCloudComponent();
+        const groupCloudComponent = new GroupCloudComponent();
+        const identityService: Identity = new Identity();
+        const groupIdentityService: GroupIdentity = new GroupIdentity();
+
+        let silentLogin;
+        let apsUser;
+        let activitiUser;
+        let noRoleUser ;
+        let selectedPeople;
+        let groupAps;
+        let groupActiviti;
+        let groupNoRole;
+        let selectedGroups;
+
+        beforeAll(async () => {
+            await identityService.init(TestConfig.identity.adminUser, TestConfig.identity.adminPassword);
+            apsUser = await identityService.createIdentityUser();
+            await identityService.assignRole(apsUser.id, CONSTANTS.ROLES.APS_USER);
+            activitiUser = await identityService.createIdentityUser();
+            await identityService.assignRole(activitiUser.id, CONSTANTS.ROLES.ACTIVITI_USER);
+            noRoleUser = await identityService.createIdentityUser();
+            selectedPeople = [`${apsUser.firstName}` + ' ' + `${apsUser.lastName}`, `${activitiUser.firstName}` + ' ' + `${activitiUser.lastName}`,
+                `${noRoleUser.firstName}` + ' ' + `${noRoleUser.lastName}`];
+            await groupIdentityService.init(TestConfig.identity.adminUser, TestConfig.identity.adminPassword);
+            groupAps = await groupIdentityService.createIdentityGroup();
+            await groupIdentityService.assignRole(groupAps.id, CONSTANTS.ROLES.APS_ADMIN);
+            groupActiviti = await groupIdentityService.createIdentityGroup();
+            await groupIdentityService.assignRole(groupActiviti.id, CONSTANTS.ROLES.ACTIVITI_ADMIN);
+            groupNoRole = await groupIdentityService.createIdentityGroup();
+            selectedGroups = [`${groupAps.name}` , `${groupActiviti.name}`, `${groupNoRole.name}`];
+            silentLogin = false;
+            settingsPage.setProviderBpmSso(TestConfig.adf.hostBPM, TestConfig.adf.hostSso, TestConfig.adf.hostIdentity, silentLogin);
+            loginSSOPage.clickOnSSOButton();
+            loginSSOPage.loginAPS(TestConfig.adf.adminEmail, TestConfig.adf.adminPassword);
+            navigationBarPage.navigateToPeopleGroupCloudPage();
+            peopleGroupCloudComponentPage.checkGroupsCloudComponentTitleIsDisplayed();
```

Comment:
Lines 74-75 are not needed here since you have them in the beforeEach()

Reply:
done